### ISSUE TITLE
IP address changes after save in control panel

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#959]
**Description**
After a submission is saved in the control panel, the IP address of the original submitter is replaced with the IP address of the user saving the submission in the control panel.

**Steps to reproduce**
1. Navigate to the CP edit page for a submission created by a user outside your network. Make note of the IP address in the meta section.
2. Click Save. After page reload the IP address shows that of the user saving the submission in the Craft CP.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.6.1
- Craft version: 3.7.45.2
- Multi-site: Yes

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.2](https://github.com/verbb/formie/releases/tag/1.6.2)
